base/script/protobuf.gradle

```diff
@@ -31,8 +31,24 @@ dependencies {
     protobuf deps.build.protobuf
 }
 
+ext {
+    compiledProtoRoot = "$projectDir/generated"
+    mainCompiledProto = "$compiledProtoRoot/main/java"
+    testCompiledProto = "$compiledProtoRoot/test/java"
+}
+
+final def mainPruneTask = task pruneGoogleProtos(type: Delete) {
+    delete "$mainCompiledProto/com"
```

Comment:
I think it's safer to be more specific here. I.e. if we mean both Google Protobuf and gRPC, let's have two.

Also, I would extract the package prefix somewhere — since you have `com` listed more than once. That would also be a good place to comment on the relation between the prefixes and the file origin.